src/components/Offramp/Confirm.view.tsx

```
@@ -301,6 +303,52 @@ export const OfframpConfirmView = ({
+            // fetch route details using link details to calculate slippage
+            const { route } = await utils.fetchRouteRaw(
+                preparedCreateLinkWrapperResponse?.linkDetails.tokenAddress ?? '',
+                preparedCreateLinkWrapperResponse?.linkDetails.chainId ?? '',
+                usdcAddressOptimism,
+                optimismChainId,
+                preparedCreateLinkWrapperResponse?.linkDetails.tokenDecimals ?? 0,
+                String(preparedCreateLinkWrapperResponse?.linkDetails.tokenAmount ?? '0')
+            )
+
+            // get slippage percentage from route details
+            if (route.params && route.params.slippage) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)